### DISCUSSION TITLE
What are the numbers in the first three rows of the first column in the CAMELS US forcing txt files?

### DISCUSSION THREAD

**Discussion by darylsuen** [neuralhydrology/neuralhydrology#185] [Q&A]
Could someone please tell me what the number are in the first three rows of the first column in the CAMELS US forcing txt files? I thinik the third number is basin area.
Thanks.

**Comment by kratzert** [comment]
Hi @darylsuen 

thanks for your enthusiasm using NeuralHydrology in the recent days/weeks. That being said, I was wondering how much time you put into solving some of these questions yourself before coming here and asking for help. Your question is a) not related to NeuralHydrology and b) can be resolved within 1 minute by looking at the CAMELS dataset. There is literally a file being called `readme_basin_mean_forcing.txt`  in the root directory of the CAMELS dataset that would answer your questions.

I appreciate activity here and questions around the use of NeuralHydrology, but I also feel like you should be respectful with the time of others and show some effort into solving a question on your own.

**Reply by darylsuen** [reply]
I apologize. Yes I am pouring over the documentation but there is a lot of it and I'm new to solving these errors so I miss things. I work 112 hours per week and am very time limited so I fully understand your frustration with my questions. I will bother you no further.
